src/transport_picoquic.cpp

```diff
@@ -847,9 +845,8 @@ PicoQuicTransport::client(const TransportContextId tcid)
             return;
         }
 
-        ret = picoquic_packet_loop(quic_ctx, 0, AF_INET, 0, 2000000, 0, pq_loop_cb, this);
+        ret = picoquic_packet_loop(quic_ctx, 0, PF_UNSPEC, 0, 2000000, 0, pq_loop_cb, this);
```

Comment:
Support IPv6 and IPv4!!!

Reply:
For the server (accepting connections), it would need to listen for packets on both IPv4 and IPv6 (thus, using two file descriptors). Does it do that?